README.md

```diff
@@ -51,7 +54,23 @@ An example project to show one use case or workflow for creating "variants".  Th
 	* Build all bundles
 	* Remove the extra files/groups that were created.
   * After building with "Pack Variations", you can enter play mode using the standard "Packed Play Mode" script.
- 
+   
+#### *Advanced/Sync Addressables*
+Synchronous Addressables!  What a crazy thing.  This sample shows how you could set up your project to utilize Addressables, but in a (nearly) completely synchronous way.
+Why don't we put these sync methods in Addressables itself?  The best way to understand that is to look at SyncAddressables/SyncAddressables.cs and search for "throw".  The code is very specific about how it needs to be used, and will cause pain for the caller if not used in the right way at the right time.  That being said, if you want to create a game built on sync interfaces, you can copy this code, and run with it.  If you are using it, all the existing async methods would still work, so you are capable of doing a mix & match in your game, if you are willing to accept the constraints when diong things sync. Note that the group schema is what associates a given asset group with either the sync providers or the regular ones.  So you could not mix & match within a group.
```

Comment:
```suggestion
Why don't we put these sync methods in Addressables itself?  The best way to understand that is to look at SyncAddressables/SyncAddressables.cs and search for "throw".  The code is very specific about how it needs to be used, and will cause pain for the caller if not used in the right way at the right time.  That being said, if you want to create a game built on sync interfaces, you can copy this code, and run with it.  If you are using it, all the existing async methods would still work, so you are capable of doing a mix & match in your game, if you are willing to accept the constraints when doing things sync. Note that the group schema is what associates a given asset group with either the sync providers or the regular ones; so you could not mix & match within a group.
```

Reply:
wow that didn't work great at all.  Basically there's a typo in "when diong things" (doing).  And the last sentence I think should be combined with the previous one (I've added a semi-colon but maybe rewording it would be better, I'm not sure).